ibis/pandas/tests/test_operations.py

```diff
@@ -1446,3 +1446,12 @@ def test_scalar_broadcasting(batting, batting_df):
     result = expr.execute()
     expected = batting_df.assign(demeaned=batting_df.G - batting_df.G.mean())
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize('raw_value', [0.0, 1.0])
+def test_scalar_parameter(t, df, raw_value):
+    value = ibis.param(dt.double)
+    expr = t.float64_with_zeros == value
+    result = expr.execute(params={value: raw_value})
```

Comment:
What do you think about named parameters (versus having to use the param object as a key)? Can be follow up work